src/agents/guardrail.py

```diff
@@ -127,12 +128,12 @@ async def run(
 @dataclass
 class OutputGuardrail(Generic[TContext]):
     """Output guardrails are checks that run on the final output of an agent.
-    They can be used to do check if the output passes certain validation criteria
+    They can be used to check if the output passes certain validation criteria
```

Comment:
to me, either is fine; let me revert this
```suggestion
    They can be used to do check if the output passes certain validation criteria
```